game.js

```diff
@@ -335,9 +335,11 @@ class Game {
   reset() {
     this.view = new View(this.squareSize);
     this.board = new Board(this.width, this.height);
-    this.isRunning = true;
-
+    this.isRunning = false;
+    this.generation = 0;
```

Comment:
good call on reseting the generation here. but make sure to also remove it from the game constructor. the purpose of calling the reset function in the constructor is so that you don't have to set it in two places.